SPECS/selinux-policy/selinux-policy.spec

```diff
@@ -302,6 +311,9 @@ exit 0
 selinuxenabled && semodule -nB
 exit 0
 %changelog
+* Mon Mar 14 2022 Chris PeBenito <chpebeni@microsoft.com> - 2.20220106-3
+- Additional policy fixes for enforcing core images.
```

Comment:
Would you be able to provide a bit more detail here about what the patches acheive.